### PR TITLE
Add first version of proposal for lightning talk by Guzal and Svea

### PR DESCRIPTION
**Pull request by SveaMeyer13**
Kindly review @aiwalter, @mloning, @fkiraly

### CODE REVIEW

File: conferences/PyData_Global_2021/Proposal_lightning_talk.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Proposal Lightning Talk
+
+
+Introducing sktime! Time series data is ubiquitous in many applications. Developing advanced time series analysis capabilities for researchers and practitioners is one of the major challenges of contemporary machine learning. sktime is a new unified Python toolbox for machine learning with time series. Our ambition is to provide for time series what scikit-learn provides for tabular data. In our lightning talk you will learn about the various use cases of the sktime toolbox: ranging from forecasting, to classification, anomaly detection and clustering of time series data.
```
Comment (aiwalter) [inline conferences/PyData_Global_2021/Proposal_lightning_talk.md:4]:
```suggestion
Introducing sktime, a unified interface for machine learning with time series. Time series data is ubiquitous in many applications. Developing advanced time series analysis capabilities for researchers and practitioners is one of the major challenges of contemporary machine learning. sktime is a new unified Python toolbox and our ambition is to provide for time series what scikit-learn provides for tabular data. In our lightning talk you will learn about the various time series learning problems covered by sktime, which are forecasting, classification/regression, annotation and clustering.
```

**Review by mloning — COMMENTED** [review]
I think this is good! Really happy that you want to submit this! 

Perhaps I would: 
* rephrase "Introducing sktime, ..." to "We present sktime, ..." since sktime has been presented before
* rephrase "you will learn about" to "we give an overview of" because it's a short talk 

In terms of the content, I would want to give an overview about (i) why we need sktime (currently fragmented ecosystem -> unified framework), (ii) what sktime is/offers (like sklearn, modular, composable, time series algorithms, multiple tasks) and (iii) how to get involved.

**Comment by mloning** [comment]
@GuzalBulatova @SveaMeyer13 if you want to push the version that you submitted we could merge the PR. It will be useful to have something to look at for the next time we want to apply!

**Comment by SveaMeyer13** [comment]
Thanks for all your feedback @mloning and @aiwalter! We've updated the proposal accordingly. It is now ready to be merged.

**Review by mloning — APPROVED** [review]

**Pull request merged**